src/components/ItemAccordion/ItemAccordion.test.js

```diff
@@ -41,4 +48,37 @@ describe('ItemAccordion', () => {
     expect(screen.getByRole('group', { name: /electronic access/i })).toBeInTheDocument();
     expect(screen.getByRole('button', { name: /add electronic access/i })).toBeInTheDocument();
   });
+
```

Comment:
Consider wrapping all these tests in a `describe()` block with a label like 'Barcode generator'. Some of the `it` labels are confusing, e.g. "... when settings are off". Settings for what?